doc/example-spnavrc

```diff
@@ -75,6 +75,8 @@
 # available actions:
 #    none,
 #    sensitivity-up, sensitivity-down, sensitivity-reset
+#    filter-rotation
```

Comment:
I'm not sure "filter" conveys what this is supposed to do.  Filter to me implies "changing" something, not completely blanking it. Like filtering to remove input jitter or noise. I think "mask" is a better verb for what you're trying to do here. What do you think? So "mask-rotation" and "mask-translation". This goes for every other instance of filter in the code of course.